io.frappe.books.yml

```diff
@@ -0,0 +1,46 @@
+app-id: io.frappe.books
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  # - --share=network
+  # only access the app's dir on the host
+  - --filesystem=~/Frappe Books:create
+build-options:
+  append-path: /usr/lib/sdk/node20/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: books
+    buildsystem: simple
+    build-commands:
+      # create an app dir in the flatpak's bin
+      - mkdir -p /app/bin/books
+      # move the contents from the builddir to flatpak's filesystem
+      - mv /run/build/books/'Frappe Books'/* /app/bin/books/
+      # Install app wrapper
+      - install -Dm755 /run/build/books/run.sh /app/bin/
+      # Install desktop icon
+      - install -Dm644 /run/build/books/share/icons/hicolor/256x256/apps/frappe-books.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      # install the desktop file
+      - install -Dm644 ${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: "https://github.com/frappe/books/releases/download/v0.23.0/frappe-books-0.23.0.x86_64.rpm"
```

Comment:
why don't you build it from source?

Reply:
I have tried. But npm install fails with an error (after generating the sources list from flatpak-builder-tools)

Reply:
Many thanks for the insighful comments. I have learnt a lot more from this PR

This is my first flatpak build & flathub submissions and i greatly appreciate your guidance :)